extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/controlBar.js

```diff
@@ -0,0 +1,506 @@
+/********************************************************************
+  CONTROL BAR
+*********************************************************************/
+var React = require('react'),
+    ReactDOM = require('react-dom'),
+    CONSTANTS = require('../constants/constants'),
+    ClassNames = require('classnames'),
+    ScrubberBar = require('./scrubberBar'),
+    Slider = require('./slider'),
+    Utils = require('./utils'),
+    Popover = require('../views/popover'),
+    VideoQualityPanel = require('./videoQualityPanel'),
+    ClosedCaptionPopover = require('./closed-caption/closedCaptionPopover'),
+    Logo = require('./logo');
+    Icon = require('./icon');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "Icon" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)